### PR TITLE
Add/jetpack growth to constants

### PR DESCRIPTION
**Pull request by CodeyGuyDylan**
## Proposed changes:

* Add Jetpack Growth constants so the My Plan section displays the new bundle

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

P2: pbNhbs-bxX-p2

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout the store sandbox on your sandbox in wpcom
2. Go to https://wordpress.com/checkout/jetpack/jetpack_growth_yearly and purchase the product
3. Start up a JN site with sandbox access active and activate your sandbox on the site
4. Go to `/wp-admin/admin.php?page=my-jetpack#/add-license` and Activate the growth license
image
5. Go to `/wp-admin/admin.php?page=jetpack#/my-plan` and ensure that the Growth plan loads with no errors or wonky UI
image

### CODE REVIEW

File: projects/plugins/jetpack/_inc/client/lib/plans/constants.js [hunk 1]
```diff
@@ -117,6 +120,7 @@ export const JETPACK_MONTHLY_PLANS = [
 	PLAN_JETPACK_SECURITY_T1_MONTHLY,
 	PLAN_JETPACK_SECURITY_T2_MONTHLY,
 	PLAN_JETPACK_COMPLETE_MONTHLY,
+	PLAN_JETPACK_COMPLETE_MONTHLY,
```
Comment (elliottprogrammer) [inline projects/plugins/jetpack/_inc/client/lib/plans/constants.js]: This should be `PLAN_JETPACK_GROWTH_MONTHLY`, right?
```suggestion
	PLAN_JETPACK_GROWTH_MONTHLY,
```

**Review by elliottprogrammer — CHANGES_REQUESTED** [review]
Tested, works good.  However there appears to be a little issue in the monthly plan constants code (See inline comments).

File: projects/plugins/jetpack/_inc/client/lib/plans/constants.js (hunk 1 above)
Comment (CodeyGuyDylan) [reply projects/plugins/jetpack/_inc/client/lib/plans/constants.js]: Yes it should 😅 thank you for catching that!

**Review by elliottprogrammer — DISMISSED** [review]
LGTM! 👍

**Review by elliottprogrammer — APPROVED** [review]

**Pull request merged**